### DISCUSSION TITLE
Ubuntu下上传本地磁盘文件io占用极高

### DISCUSSION THREAD

**Discussion by dahetaoa** [AlistGo/alist#5382] [Show and tell]
![Screenshot_20231015-141428369](https://github.com/alist-org/alist/assets/102645524/edc87472-f17f-4639-8629-7c80b9bb8f2a)
我的配置环境是Ubuntu下用自动脚本安装了alist，Ubuntu内挂载了一个外接硬盘，我使用alist的本地存储挂载了我的硬盘，但是在上传文件到硬盘的时候，明明上传完成了，但过了很久，cpu的io等待还是非常高，并且集中在一个核心上，具体现象是我的Ubuntu有4个核，其中一个核的io等待高达95%，而这个核心的系统占用却很少，别的核心的占用和io等待也很少，查看磁盘写入，发现几乎没有磁盘正在写入中，有也是很小的速度，所以我怀疑是alist的某个bug导致硬盘实际上没有在读/写，但是还是会占用cpu导致高额的io等待，希望能得到解决😢

**Comment by iwvw** [comment]
> ![Screenshot_20231015-141428369](https://user-images.githubusercontent.com/102645524/275306523-edc87472-f17f-4639-8629-7c80b9bb8f2a.jpg) 我的配置环境是Ubuntu下用自动脚本安装了alist，Ubuntu内挂载了一个外接硬盘，我使用alist的本地存储挂载了我的硬盘，但是在上传文件到硬盘的时候，明明上传完成了，但过了很久，cpu的io等待还是非常高，并且集中在一个核心上，具体现象是我的Ubuntu有4个核，其中一个核的io等待高达95%，而这个核心的系统占用却很少，别的核心的占用和io等待也很少，查看磁盘写入，发现几乎没有磁盘正在写入中，有也是很小的速度，所以我怀疑是alist的某个bug导致硬盘实际上没有在读/写，但是还是会占用cpu导致高额的io等待，希望能得到解决😢

请问这是什么软件？
